sphinx/source/docs/user_guide/interaction/widgets.rst

```diff
@@ -166,6 +166,14 @@ A widget for collecting a line of text from a user:
 .. bokeh-plot:: docs/user_guide/examples/interaction_textinput.py
     :source-position: below
 
+TextareaInput
+~~~~~~~~~
```

Comment:
We have strict sphinx options turned on, this causes an error in the build. The underline length needs to match the title length exactly.